notebooks/3_KernelMethods.ipynb

```diff
@@ -623,11 +627,11 @@
    "outputs": [],
    "source": [
     "regularizations = np.linspace(-10, -2, 24)\n",
-    "krrs = [KRR(regularization=10**i, kernel_type=kernel_type) \n",
+    "krrs = [KernelRidge(alpha=10**i, kernel = 'rbf', gamma = 1.0) \n",
```

Comment:
You shouldn't change the kernels in *each* KRR constructor. The point of using a `kernel_type` variable is to allow users to change one variable at the top of the page and be able to compare results across the whole notebook. I would suggest changing linears 102-103 from

```   
kernel_func = gaussian_kernel
kernel_type = 'gaussian'
```

to 
```
kernel_params = {"kernel": "rbf", "gamma": 1.0}
```

and then using `**kernel_params` in each of the KRR constructors, as shown below. This will allow someone to then define their own kernel.

```suggestion
    "krrs = [KernelRidge(alpha=10**i, **kernel_params) \n",
```

Reply:
Okay.